### DISCUSSION TITLE
The parameter show_icons  affects the number of statistics

### DISCUSSION THREAD

**Discussion by keeStep** [anuraghazra/github-readme-stats#3105] [Q&A] [answered]
### Describe the bug

The number of rows displayed when show_icons are added is different from if show_icons are not added

![image](https://github.com/anuraghazra/github-readme-stats/assets/38199712/09e6651d-0274-447d-a20a-51c497100577)

**Comment by qwerty541** [comment, marked as answer]
`show_icons` query option cannot affect stats numbers. I just checked your card and it shows same stats in both cases.

Most likely this behavior was caused by caching. As noticed in documentation we use caching to decrease the load on our servers (see https://github.com/anuraghazra/github-readme-stats/issues/1471#issuecomment-1271551425). Our cards have a default cache of 4 hours (14400 seconds). Looks like your card was cached and when you tried to enable icons the new one was generated with latest stats.

<img src="https://github-readme-stats.vercel.app/api?username=keeStep" />

<img src="https://github-readme-stats.vercel.app/api?username=keeStep&show_icons=true" />

**Reply by rickstaa** [reply]
@Aein29 although there is nothing wrong with, friendly hellos. Can you please stop spamming this repository, since me and the other collaborators get notified every time you do? I really don't want to have to report you to GitHub 😅.
